test/cmdline_test.py

```diff
@@ -125,12 +125,12 @@ def test_cmdline_local_scheduler(self, logger):
 
     @mock.patch("logging.getLogger")
     def test_cmdline_other_task(self, logger):
-        luigi.run(['--local-scheduler', '--no-lock', 'SomeTask', '--n', '1000'])
+        luigi.run(['SomeTask', '--local-scheduler', '--no-lock', '--n', '1000'])
```

Comment:
This shouldn't be needed to change right?

Reply:
✔︎